src/components/bottom-sheet/BottomSheetContainer.tsx

```diff
@@ -1,15 +1,27 @@
 import React, {ReactNode} from 'react';
+import {ViewStyle} from 'react-native';
 import {useWindowDimensions, View} from 'react-native';
 
+export enum BottomSheetSize {
+  auto = 0,
+  compact = 0.5,
+  cover = 0.8,
+}
```

Comment:
Can't we just have a parameter `maxHeight` as a number parameter instead? I think this is more intuitive to understand when developing.

Reply:
It is fixed to just use `max-height value` as a per cent value of the screen.

Reply:
Fixed